webapp/lib/view.dart

```diff
@@ -535,13 +504,27 @@ class BatchRepliesConfigurationView extends PackageConfiguratorView {
     );
   }
 
-  void _addTagDropdown(MouseEvent event, List<String> tags) {
-    var tagsList = (event.target as Element).parent;
-    if (tagsList.children.last.classes.contains('add-tag-dropdown')) tagsList.lastChild.remove();
+  SpanElement _addTagAction(List<String> tags) {
+    return new SpanElement()
+      ..classes.add('tags__actions')
+      ..append(
+        new ButtonElement()
+          ..classes.add('button-add-tag')
+          ..text = '+'
+          ..onClick.listen((event) {
+            _addTagDropdown(event, model.tags);
+            event.stopPropagation();
+          })
+      );
+  }
 
+  void _addTagDropdown(MouseEvent event, List<String> tags) {
+    var tagActions = (event.target as Element).parent;
+    var tagsList = tagActions.parent;
+    document.querySelectorAll('.add-tag-dropdown').forEach((dropdown) => dropdown.remove());
```

Comment:
this might not be needed following https://github.com/larksystems/nook-configurator/pull/22/files#r510254754

Reply:
Corrected